docs/notebooks/icnn_inits.ipynb

```diff
@@ -0,0 +1,557 @@
+{
```

Comment:
<p>The Gaussian approximation-based initialization schemes require samples from both, source and target distributions, in order to initialize the ICNNs with linear factors and means, as detailed in <a href="https://arxiv.org/pdf/2206.14262.pdf" target="_blank">Bunne et al. (2022)</a>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/90/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
<p>Changed this.</p><p> </p>